jekyll/_cci2/server/v4.3/installation/phase-1-prerequisites.adoc

```diff
@@ -94,13 +94,13 @@ CircleCI server installs into an existing Kubernetes cluster. The application us
 | NIC speed
 
 | < 500
-| 3
-| 12 cores
-| 32 GB
+| 4
```

Comment:
Do we need to update these commands on creating the clusters?
1. https://circleci.com/docs/server/v4.3/installation/phase-1-prerequisites/#eks
2. https://circleci.com/docs/server/v4.3/installation/phase-1-prerequisites/#gke